packages/core/src/error/IndySdkError.ts

```diff
@@ -0,0 +1,11 @@
+import type { IndyError } from '../utils/indyError'
+
+import { AriesFrameworkError } from './AriesFrameworkError'
+
+export class IndySdkError extends AriesFrameworkError {
```

Comment:
This is kind of interesting. I would assume that `IndySdk` is lower-level detail than framework level error. Therefore, I'm not sure about `IndySdkError` extending `AriesFrameworkError`

Reply:
The normal `IndyError` is lower-level detail than the framework level error. However, because the normal error is not very nice in stacktracing/information we've wrapped it with a framework specific error.

To me `AriesFrameworkError` is the basis for all framework created errors (which is the case if we wrap the original error). I think it can be beneficial to have a single base error that allows to catch all framework errors. Maybe AriesFrameworkError is not the best name as the base error